app/assets/scripts/views/donate.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+import React from 'react';
+
+import createReactClass from 'create-react-class';
+
+import ConnectFold from '../components/connect-fold';
+
+import YoutubeEmbed from '../components/youtube-player';
+
+/*
+ * create-react-class provides a drop-in replacement for the outdated React.createClass,
+ * see https://reactjs.org/docs/react-without-es6.html
+ * Please modernize this code using functional components and hooks!
+ */
+var Donate = createReactClass({
+  displayName: 'Donate',
+
+  propTypes: {},
+
+  render: function () {
+    return (
+      <section className="inpage">
+        <header className="inpage__header">
+          <section className="fold fold--type-a" id="donate-fold-story">
+            <div className="inner">
+              <header className="fold__header donate-window">
+                <div>
+                  <h1 className="fold__title">Donate now to OpenAQ!</h1>
+                  <div className="prose prose--responsive">
+                    <p>
+                      Help support our mission to end air pollution through air
+                      quality data and community. Donate below or directly with{' '}
+                      <a
+                        href="https://www.dafdirect.org/DAFDirect/daflink?_dafdirect_settings=NDc1MzI0MTcyXzIxMTFfZDA0ODQ3MzYtOGJkOS00YTJjLWFmOTctM2Q2MGE3YTg4MDU4&designatedText=Z2VuZXJhbCBvcmdhbml6YXRpb25hbCBzdXBwb3J0ICh1bnJlc3RyaWN0ZWQp&amountValue="
```

Comment:
@sruti please confirm the correct link we will want for DAF

Reply:
This is correct